src/Components/OATGraphViewer/Internal/OATGraphCustomEdge.tsx

```diff
@@ -154,9 +158,10 @@ const OATGraphCustomEdge: React.FC<IOATGraphCustomEdgeProps> = ({
         showRelationships,
         showInheritances,
         showComponents,
+        currentNodeIdRef,
         state
     } = useContext(ElementsContext);
-    const { model } = state;
+    const { model, selectedModelId } = state;
```

Comment:
Do model and selectedModelId always get updated in tandem? I.e. model[@id] == selectedModelId? If so, we should get rid of the latter.